go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -392,6 +397,45 @@ func expectDeleteQueries(t *testing.T) {
 	})
 }
 
+func expectLogsAndUnsubscribe(t *testing.T, logs []LogExpectation, logCh chan interface{}) {
+	defer vrLogStatsLogger.Unsubscribe(logCh)
+	t.Helper()
```

Comment:
nit: t.Helper should be the first statement. Basically, it's as if it's part of the function declaration.